google-cloud-spanner/pom.xml

```diff
@@ -60,7 +60,8 @@
         <executions>
           <execution>
             <goals>
-              <goal>integration-test</goal>
+              <!--<goal>integration-test</goal>-->
+              <goal>verify</goal>
```

Comment:
I realize it adds dirt, but having to manually update it is quite disruptive; people who are used to the no-touch commands now would likely forget to update it.